### DISCUSSION TITLE
Calculated total of  dynamically created input price field

### DISCUSSION THREAD

**Discussion by abdulmateentechbits** [react-hook-form/react-hook-form#9196] [Q&A] [answered]
How can we calculate total dynamically created field,
I have location details where we have amount field when user checked on fixed then amount field is shown i want to calculate the total as user created new location the amount field will be add to the state 
https://codesandbox.io/s/property-form-updated-forked-b8cwu3

@leapful 
@bluebill1049

**Comment by bluebill1049** [comment, marked as answer]
have you watched this video tutorial? https://www.youtube.com/watch?v=4MrbfGSFY2A&feature=emb_logo i think it has the answer for you.

**Reply by abdulmateentechbits** [reply]
@bluebill1049 Thankyou so much
